packages/website/utils/contentfulSidebarLinks.json

```diff
@@ -8,6 +8,10 @@
       "title": "Adding icons",
       "slug": "/guidelines/adding-icons"
     },
+    {
+      "title": "Forma 36 component statuses",
```

Comment:
Maybe it should be called differently? Something like "Components release statuses"? WDYT?

Reply:
I would just call it `Component release phases` without Forma 36. Since you are already on the f36 website, you know that it's related to forma